evaluate.py

```diff
@@ -171,14 +170,31 @@ def run_evaluation_episode(trace_name, model, world, camera, car, video_dir, sav
 
     return run_metrics
 
+
+def create_steering_model(args):
+    if args.model_type == "steering":
+        model = SteeringModel(args.model)
+    elif args.model_type == "conditional-steering":
+        model = ConditionalSteeringModel(args.model, args.speed_model)
+    elif args.model_type == "conditional-waypoints":
+        model = ConditionalWaypointsModel(args.model, args.speed_model)
+    else:
+        print(f"Uknown model type {args.model_type}")
+        sys.exit()
+
+    return model
+
+
 if __name__ == '__main__':
 
     run_start_time = int(time.time())
 
     parser = argparse.ArgumentParser()
     parser.add_argument('--wandb-project', type=str, help='Weights and Biases project for logging.')
     parser.add_argument('--save-video', action='store_true', help='Save video of model run.')
-    parser.add_argument('--model', type=str, required=True, help='Path to ONNX model.')
+    parser.add_argument('--model', type=str, required=True, help='Path to ONNX model to predict vehicle steering.')
+    parser.add_argument('--speed-model', type=str, required=False, help='Path to ONNX model to predict vehicle speed.')
+    parser.add_argument('--model-type', type=str, required=False, default="steering", help="Model type to use for making predictions.")
     parser.add_argument('--resize-mode', default='resize', choices=['full', 'resize'], help='Resize mode of the input images (bags pre-processed for Vista).')
     parser.add_argument('--dynamics', default='learned', choices=['learned', 'exp-mov-avg', 'none'], help='Which vehicle dynamics model to use. Defaults to a learned 10hz GRU model.')
```

Comment:
"Vehicle dynamics model" is not correct terminology here, rather it's "vehicle latency model".

Reply:
This is not related to this PR.